### PR TITLE
Improvements and bugfixes

### PR DESCRIPTION
**Pull request by petrparolek**
- upgraded PHPStan
- downgraded ninjify/coding-standard

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -44,9 +44,9 @@
     "nette/tester": "^2.4.0",
     "tracy/tracy": "^2.8.0",
     "mockery/mockery": "^1.3.0",
-    "phpstan/phpstan": "^0.12",
-    "phpstan/phpstan-strict-rules": "^0.12",
-    "ninjify/coding-standard": "^0.12"
+    "phpstan/phpstan": "^1.0",
+    "phpstan/phpstan-strict-rules": "^1.0",
+    "ninjify/coding-standard": "^0.11"
```
Comment (f3l1x) [inline composer.json:49]: Proc 0.11?

File: Makefile [hunk 2]
```diff
@@ -12,7 +12,7 @@ csf:
 	vendor/bin/phpcbf --standard=ruleset.xml --extensions=php,phpt --tab-width=4 --ignore=temp -sp src tests
 
 phpstan:
-	vendor/bin/phpstan analyse -l max -c phpstan.neon --memory-limit=512M src
+	vendor/bin/phpstan analyse -l 8 -c phpstan.neon --memory-limit=512M src
```
Comment (f3l1x) [inline Makefile:15]: Presunes to do configu, prosim?

File: composer.json (hunk 1 above)
Comment (petrparolek) [reply composer.json:49]: opravím

File: Makefile (hunk 2 above)
Comment (petrparolek) [reply Makefile:15]: OK

File: composer.json (hunk 1 above)
Comment (petrparolek) [reply composer.json:49]:
```
root@7c4d2bf04b30:~/repos/contributte-datagrid-nette-database-data-source# make qa
vendor/bin/phpstan analyse -l 8 -c phpstan.neon --memory-limit=512M src
 3/3 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


                                                                                                                        
 [OK] No errors                                                                                                         
                                                                                                                        

vendor/bin/phpcs --standard=ruleset.xml --extensions=php,phpt --tab-width=4 --ignore=temp -sp src tests
ERROR: Referenced sniff "SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameAfterKeyword" does not exist

Run "phpcs --help" for usage information

make: *** [Makefile:9: cs] Error 3
```

Comment (f3l1x) [reply composer.json:49]: Odmaz FullyQualifiedClassNameAfterKeyword z ruleset.xml a pripadne dalsi co delaji problemy.

File: Makefile [hunk 3]
```diff
@@ -12,7 +12,7 @@ csf:
 	vendor/bin/phpcbf --standard=ruleset.xml --extensions=php,phpt --tab-width=4 --ignore=temp -sp src tests
 
 phpstan:
-	vendor/bin/phpstan analyse -l max -c phpstan.neon --memory-limit=512M src
+	vendor/bin/phpstan analyse -c phpstan.neon --memory-limit=512M src
```
Comment (f3l1x) [inline Makefile:15]: Path take do neonu

File: composer.json (hunk 1 above)
Comment (petrparolek) [reply composer.json:49]: jak? Vždyt toto tam není

File: Makefile (hunk 3 above)
Comment (petrparolek) [reply Makefile]: OK

**Pull request merged**